single_tech_samples/databricks/sample4_ci_cd/notebook-python-lib/README.md

```diff
@@ -1 +1,49 @@
-# Readme
+# Testing a Databricks Python Notebook with imported custom libraries
+
+## Overview
+This sample demonstrates:
+1. how to import custom python libraries/modules into a databricks notebook
+2. how to use unit tests to test the python modules and use [nutter](https://github.com/microsoft/nutter) to perform integration test on the notebook
+3. how to package the python modules & notebook, execute tests and publish the results as part of Build and Release pipelines
+
+## How to use this sample
+### Prerequisites
+
+- [Azure DevOps account](https://azure.microsoft.com/en-au/services/devops/) for running pipelines
+- [Github account](https://github.com/) for source control
+- [Azure Databricks workspace](https://azure.microsoft.com/en-au/services/databricks/) with an [existing cluster](https://docs.microsoft.com/en-us/azure/databricks/clusters/create) running. Note: Refer to [Sample 1](../../sample1_basic_azure_databricks_environment/README.md) and [Sample 3](../../sample3_cluster_provisioning_and_data_access/README.md) to setup these resource.
+
+### Setup and deployment
+
+1. Clone (or import) this repository.
+1. In Azure DevOps, [define a Variable Group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml) called `mdwdo-dbx-npl-test` with the following variables:
+    1. **databricksDomain** - The databricks host in this format: `https://adb-123456789xxxx.x.azuredatabricks.net`
+    1. **databricksToken** - Databricks PAT token.
+    1. **databricksNotebookPath** - Where in the databricks workspace the notebook will be uploaded as part of the Release pipeline. ei. `/Shared/notebooks`
```

Comment:
Do we want to add `databricksLibsPath`? Or just hardcode for now to dbfs:/FileStore/jars?

Reply:
@devlace Thanks. I've changed it to use databricksLibPath instead.